TestFoundation/TestMeasurement.swift

```diff
@@ -8,7 +8,7 @@
 //
 
 class CustomUnit: Unit {
-    required init(symbol: String) {
+    required override init(symbol: String) {
```

Comment:
We need a bug for this.

Reply:
I think the correct fix here is to remove the `required` on `Unit.init(symbol: String)` to match Darwin:
https://developer.apple.com/documentation/foundation/unit/1690760-init
Then this `override` is required in both cases.

However removing the `required` leads to this line
`unit = UnitType(symbol: symbol)` 
in `Measurement.init(from decoder: Decoder)` giving the error:

`Constructing an object of class type 'UnitType' with a metatype value must use a 'required' initializer`

Reply:
Filed https://bugs.swift.org/browse/SR-10904